tools/releases/dockerfiles/Dockerfile.kuma-init

```diff
@@ -1,6 +1,29 @@
-FROM docker.io/istio/proxy_init:1.1.17
+FROM ubuntu:bionic as builder
+
+ENV GOVERSION 1.14.2
+ENV GOROOT /opt/go
+ENV GOPATH /root/.go
+
+RUN apt-get update
+RUN apt-get -y install wget
+
+RUN cd /opt && wget https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.tar.gz && \
+    tar zxf go${GOVERSION}.linux-amd64.tar.gz && rm go${GOVERSION}.linux-amd64.tar.gz && \
+    ln -s /opt/go/bin/go /usr/bin/ && \
+    mkdir $GOPATH
+
+RUN GO111MODULE=on go get istio.io/istio/tools/istio-iptables@1.6.5
```

Comment:
this is only run once when we build the image, right?

Reply:
Ok, one thing I don't like about this docker.
I think pretty soon we will be testing iptables on Universal (today's message on Slack). I think a safer bet is to create our kuma-init binary (just a main.go that imports it and execute) and release it like any other binary. This way we can try to use it on bare metal without docker.

Reply:
Discussed that kuma-init binary is a next big step, so this PR will provide only docker image